layercatalog.json

```diff
@@ -1272,5 +1272,56 @@
         "native_crs": "EPSG:4326"
       }
     }
+  },
+  {
+    "id": "COP_DEM_GLO_30M_COG",
+    "name": "COP_DEM_GLO_30M_COG",
+    "title": "Copernicus DEM 30M COG",
+    "description": "Copernicus Global 30 meter Digital Elevation Model dataset in COG format.",
+    "license": "proprietary",
+    "providers": [
+      {
+          "name": "VITO, on behalf of the Belgian Science Policy Office (BELSPO).",
+          "roles":
+          [
+              "producer",
+              "licensor"
+          ],
+          "url": "https://terrascope.be"
+      }
+      ],
+    "links": [
+      {
+        "rel": "license",
+        "href": "https://spacedata.copernicus.eu/documents/20126/0/CSCDA_ESA_Mission-specific+Annex.pdf"
+      }
+    ],
+    "extent": {
+      "spatial": {"bbox": [[-180, -90, 180, 90]]},
+      "temporal": {"interval": [["2010-12-12", null]]}
+    },
+    "cube:dimensions": {
+      "x": {"type": "spatial", "axis": "x", "step": 0.002777777777777778, "reference_system": "EPSG:4326"},
+      "y": {"type": "spatial", "axis": "y", "step": 0.002777777777777778, "reference_system": "EPSG:4326"},
+      "t": {"type": "temporal"},
+      "bands": {
+        "type": "bands",
+        "values": ["Copernicus_DSM_COG_10"]
```

Comment:
Let's use 'DEM' as band name, and use the aliases system to ensure that oscars gets the correct link title.

Reply:
Actually the complex link title in oscars is also questionable :-)

Reply:
Done.